extra/healthcheck.go

```diff
@@ -32,20 +32,20 @@ func main() {
 	}
 
 	sslKey := os.Getenv("UPTIME_KUMA_SSL_KEY")
-	if len(sslKey) == 0 {
+	if sslKey == "" {
```

Comment:
Entirely unrelated change.
Please revert this file

Reply:
same thing with this as well will remove it